### PR TITLE
One should return false (is not prime)

### PR DESCRIPTION
**Pull request by gsears**
First PR ever...

Just testing everything out to see if it works. Plus I'm a stickler for mathematical purity... ;) 

Also, it turns out my auto-formatter has formatted the code...

Should these technically be two PRs?

Cheers!

### CODE REVIEW

File: src/main/java/compsci4039/week1/loops/Lab4Ex2.java [hunk 1]
```diff
@@ -31,6 +31,8 @@ private static void calculatePrimeNumbersUntil(int number) {
     private static boolean isPrime(int number) {
         if (number < 0) {
             return false;
+        } else if (number == 1) {
+            return false;
```
Comment (BelfDev) [inline src/main/java/compsci4039/week1/loops/Lab4Ex2.java]: Hey @gsears!

Excellent catch. Indeed, I forgot about number 1. I would suggest modifying the first conditional clause tho, don't you think ```if(number < 2) return false``` would be more elegant?

File: src/main/java/compsci4039/week1/loops/Lab4Ex2.java [hunk 2]
```diff
@@ -62,13 +64,15 @@ private static void executeGuessingGame() {
         int numberOfAttempts = 10;
         int currentAttemptIndex = 0;
 
-        System.out.println("Hello! Welcome to my awesome guessing game.\nGuess which number I've picked. You can choose between from 0 - 99");
+        System.out.println(
+                "Hello! Welcome to my awesome guessing game.\nGuess which number I've picked. You can choose between from 0 - 99");
```
Comment (BelfDev) [inline src/main/java/compsci4039/week1/loops/Lab4Ex2.java]: We should either commit the formatting rules (e.g. the ```.prettierrc``` file) or never stage those to the addition. No problem for now as we are unlikely to further develop this class :)

**Review by BelfDev — CHANGES_REQUESTED** [review]
Great contribution, @gsears. I've indeed forgotten the number 1 case. I have suggested a few modifications tho -- see if you agree.

File: src/main/java/compsci4039/week1/loops/Lab4Ex2.java (hunk 2 above)
Comment (gsears) [reply src/main/java/compsci4039/week1/loops/Lab4Ex2.java]: Good plan! Might be worth working on some boilerplate environment stuff including a .prettierrc (I'm pretty sure it has a java plugin) as well as a gradle build for unit tests, jdoc, etc for future assignments... Might try and make one and push it to my github. Maybe we could convince some of the more experienced coders to dabble in devops...

Gradle seems like the way to go after comparing it with Maven (XML...yuck.).  

Might as well over-engineer these labs 😛

**Review by gsears — COMMENTED** [review]
See what you think 😸 

Was interesting learning how to revert back as well. Learning all the git...

**Review by BelfDev — APPROVED** [review]

**Pull request merged**